data/sql/updates/pending_db_world/rev_1632235858441614200.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632235858441614200');
+
+UPDATE 
+	`creature_loot_template` 
+SET 
+	`Chance` = 
+	(CASE 
+		WHEN `Entry` = 7438 THEN 7
+		WHEN `Entry` = 7439 THEN 7
+		WHEN `Entry` = 7440 THEN 8
+		WHEN `Entry` = 7441 THEN 8
+		WHEN `Entry` = 7442 THEN 8
+		WHEN `Entry` = 10199 THEN 7
+		WHEN `Entry` = 10916 THEN 6
+	END)
```

Comment:
what are the advantages to add this case-when instead of mulitple UPDATE lines?

Reply:
I don't know whether it is necessarily better, but my understanding is that a single query is easier to optimize for the query optimizer.